src/components/Article.jsx

```diff
@@ -7,24 +7,23 @@ import updateArticleVotes from "../assets/Utils/updateArticleVotes";
 const Article = () => {
   const { article_id } = useParams();
   const [article, setArticle] = useState({ title: "Not Found" });
-  const [count, setCount] = useState(0);
+  useEffect(() => {
+    getArticle(article_id).then((article) => {
+      setArticle(article);
+    });
+  }, []);
+  const [count, setCount] = useState(article.votes);
```

Comment:
When this line runs for the first time you are still awaiting the response from the get article call (it is asynchronous) so `article.votes` is undefined. This accounts for the value not showing up on initial render.